client/my-sites/plans-features-main/index.jsx

```diff
@@ -445,7 +508,7 @@ export class PlansFeaturesMain extends Component {
 			<div className="plans-features-main__group is-narrow">
 				<FormattedHeader
 					headerText={ translate( 'Solutions' ) }
-					subHeaderText={ translate( 'Just need backups? Learn about add-on solutions.' ) }
+					subHeaderText={ this.getProductsSelectorSubHeader() }
```

Comment:
How do you feel about moving `getProductsSelectorSubHeader` to its own component? It would really help in keeping the already huge `PlansFeaturesMain` clean from any unnecessary logic.